pyproject.toml

```diff
@@ -7,7 +7,7 @@
 [build-system]
 build-backend = "mesonpy"
 requires = [
-    "meson-python>=0.15.0",
+    "meson-python>=0.16.0",
```

Comment:
I bumped the version so that users won't face an issue and that those who are using a development edition of PyWavelets will receive a new version of `meson-python` the next time they retrieve the latest changes. What I'm not sure about is if we should do an exact pin to `0.16.0` though or if the updated lower bound is fine, i.e., in case a newer version of `meson-python` breaks the installations?

Reply:
Thiugh if it does, we can always pin it later, so I would lean towards not pinning since most users will be using the wheels anyway.

Reply:
We should never add an upper bound in `main`. In releases we do add an upper bound to dependencies that could get breaking updates, for exactly the reason you mention.